src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GithubWebTests.kt

```diff
@@ -64,4 +69,15 @@ class GithubWebTests : SeleniumTest()
                 .contains("We have not been able to successfully identify you as a member of the app's configured Github org")
 
     }
+
+    @Test
+    fun `non report readers cannot see report page`()
```

Comment:
Should there be a test that report readers can see the page as well? Oh, I see you've got one for the Montagu login though.

Reply:
Yeah this was just confirming that authorization was working for GitHub. I'll change the test name to make that clearer.